sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -474,13 +489,33 @@ export type IndexerStatus = 'unknown' | 'error' | 'running';
 // @public
 export interface IndexingParameters {
     batchSize?: number;
-    configuration?: {
-        [propertyName: string]: any;
-    };
+    // (undocumented)
```

Comment:
this is now missing a reference doc -- did the swagger drop a comment?

Reply:
Yes. They added a ```$ref``` and dropped the description here: https://github.com/Azure/azure-rest-api-specs/pull/10349/files#diff-54a85d8309f1ab82d2da096e6e430b9fL5795